lua/nvim-treesitter/parsers.lua

```diff
@@ -194,13 +194,13 @@ list.java = {
   maintainers = { "@p00f" },
 }
 
-list.kotlin = {
+--[[ list.kotlin = {
   install_info = {
     url = "https://github.com/tormodatt/tree-sitter-kotlin",
     files = { "src/parser.c" },
   },
   maintainers = { "@tormodatt" },
-}
+} ]]
```

Comment:
Is there already a link for the new kotlin parser?

Reply:
It's [here](https://github.com/Joakker/tree-sitter-kotlin), though it's definitely not ready yet